product_code/configuration_service/impl/src/main/resources/logback.xml

```diff
@@ -0,0 +1,36 @@
+<configuration>
+
+    <conversionRule conversionWord="coloredLevel" converterClass="com.lightbend.lagom.internal.logback.ColoredLevel" />
+
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <filter class="ch.qos.logback.core.filter.EvaluatorFilter">
+            <evaluator>
+                <expression>de.upb.cs.uc4.shared.client.exceptions.UC4Exception.class.isInstance(throwable)</expression>
```

Comment:
Filter only UC4NonCriticalExceptions with the new writing style

Reply:
Change is applied